commit fbc3ed1339ecfe0665e7321186c06f0f66714c60
Author: Zhou Yanjie <zhouyanjie@cduestc.edu.cn>
Date:   Fri Jan 25 02:22:15 2019 +0800

    DTS: CI20: Fix bugs in ci20's device tree.
    
    [ Upstream commit 1ca1c87f91d9dc50d6a38e2177b2032996e7901c ]
    
    According to the Schematic, the hardware of ci20 leads to uart3,
    but not to uart2. Uart2 is miswritten in the original code.
    
    Signed-off-by: Zhou Yanjie <zhouyanjie@cduestc.edu.cn>
    Signed-off-by: Paul Burton <paul.burton@mips.com>
    Cc: linux-mips <linux-mips@vger.kernel.org>
    Cc: linux-kernel <linux-kernel@vger.kernel.org>
    Cc: devicetree@vger.kernel.org
    Cc: robh+dt@kernel.org
    Cc: ralf@linux-mips.org
    Cc: jhogan@kernel.org
    Cc: mark.rutland@arm.com
    Cc: malat@debian.org
    Cc: ezequiel@collabora.co.uk
    Cc: ulf.hansson@linaro.org
    Cc: syq <syq@debian.org>
    Cc: jiaxun.yang <jiaxun.yang@flygoat.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/mips/boot/dts/ingenic/ci20.dts b/arch/mips/boot/dts/ingenic/ci20.dts
index 50cff3cbcc6d..4f7b1fa31cf5 100644
--- a/arch/mips/boot/dts/ingenic/ci20.dts
+++ b/arch/mips/boot/dts/ingenic/ci20.dts
@@ -76,7 +76,7 @@
 	status = "okay";
 
 	pinctrl-names = "default";
-	pinctrl-0 = <&pins_uart2>;
+	pinctrl-0 = <&pins_uart3>;
 };
 
 &uart4 {
@@ -196,9 +196,9 @@
 		bias-disable;
 	};
 
-	pins_uart2: uart2 {
-		function = "uart2";
-		groups = "uart2-data", "uart2-hwflow";
+	pins_uart3: uart3 {
+		function = "uart3";
+		groups = "uart3-data", "uart3-hwflow";
 		bias-disable;
 	};
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1079_fbc3ed1339ecfe0665e7321186c06f0f66714c60.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1079_fbc3ed1339ecfe0665e7321186c06f0f66714c60.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
